motionplan/cBiRRT.go

```diff
@@ -177,7 +177,19 @@ func (mp *cBiRRTMotionPlanner) rrtBackgroundRunner(
 	defer close(m1chan)
 	defer close(m2chan)
 
-	mp.logger.Debugf("running CBiRRT with start map of size %d and goal map of size %d", len(rrt.maps.startMap), len(rrt.maps.goalMap))
+	seedPos, err := mp.frame.Transform(seed)
```

Comment:
this being added because of the better logging message?

Reply:
Correct